kratos/utilities/global_pointer_utilities.h

```diff
@@ -49,6 +49,10 @@ namespace Kratos
 ///@name Kratos Classes
 ///@{
 
+class Element;   /// Forward declaration
```

Comment:
we decided long ago that we should NOT use forward declarations with the goal of avoiding includes

Reply:
In that case I need to do a lot of  heavy includes

Reply:
Solved